content/en/docs/concepts/architecture/mixed-version-proxy.md

```diff
@@ -23,7 +23,7 @@ from the upgrade process.
 This mechanism is called the _Mixed Version Proxy_. 
 
 ## Enabling the Mixed Version Proxy
-Ensure that `UnknownVersionInteroperabilityProxy` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) 
+Ensure that {{< glossary_tooltip text="MVP" term_id="mixed-version-proxy" >}} [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
```

Comment:
The original text is better; there's no feature gate named `MVP`. Also, you don't need a tooltip in the page about the concept.

Reply:
Reverted.